### PR TITLE
[Postgres] az postgres up: support local context

### PR DESCRIPTION
**Pull request by arrownj**
This PR is to add local context support for `az postgres up`.
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
Postgres

File: src/db-up/azext_db_up/_params.py [hunk 1]
```diff
@@ -58,6 +59,20 @@ def load_arguments(self, _):  # pylint: disable=too-many-locals, too-many-statem
         c.argument('version', help='Server version', default='5.7')
 
     with self.argument_context('postgres up') as c:
+        c.argument('server_name', options_list=['--server-name', '-s'], help='Name of the server.',
+                   local_context_attribute=LocalContextAttribute(
+                       name='postgres_server_name', actions=[LocalContextAction.SET], scopes=['cupertino']
```
Comment (yungezz) [inline src/db-up/azext_db_up/_params.py:64]: what's this scope for?

Comment (arrownj) [reply src/db-up/azext_db_up/_params.py:64]: `cupertino` is a standalone CLI extension, you can find it [here](https://github.com/VSChina/azure-connect). It is a client of azure connect service, which required for local context support.

Comment (yungezz) [reply src/db-up/azext_db_up/_params.py]: got it. While just thinking how to best describe it, since cupertino should just be a project name.

Comment (arrownj) [reply src/db-up/azext_db_up/_params.py]: Actually it is also the root command group name. The scope of local context must be defined with a valid command group or command name.

**Review by qianwens — APPROVED** [review]

**Review by qianwens — APPROVED** [review]

**Pull request merged**